values.yaml

```
@@ -917,6 +917,9 @@ server:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name: ""
+    # Controls whether or not a long lived API token should be created for this Service account
+    # serviceAccount.create must be equal to 'true' in order to use this feature. 
+    generateToken: false
```

I'd like us to be very explicit that this is not recommended:

```suggestion
    # Create a Secret API object to store a non-expiring token for the service account.
    # Prior to v1.24.0, Kubernetes used to generate this secret for each service account by default.
    # Kubernetes now recommends using short-lived tokens from the TokenRequest API or projected volumes instead if possible.
    # For more details, see https://kubernetes.io/docs/concepts/configuration/secret/#service-account-token-secrets
    # serviceAccount.create must be equal to 'true' in order to use this feature.
    createSecret: false
```

I've also suggested a slightly different name, but I won't pepper the accompanying suggestions through the rest of the PR to avoid spam.